homeassistant/components/plaato/strings.json

```
@@ -3,7 +3,7 @@
     "step": {
       "user": {
         "title": "Set up the Plaato Webhook",
-        "description": "Are you sure you want to set up the Plaato Airlock?"
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
```

```suggestion
        "confirm_setup": "[%key:common::config_flow::description::confirm_setup%]"
```
Key standardisation is better

---

This won't work, see commit https://github.com/home-assistant/core/pull/41622/commits/b067f7a31b142cdc02c4247f2c5356ad000a4ef4

---

That is odd I did not know that.

```
Integration plaato:
* [TRANSLATIONS] Invalid strings.json: extra keys not allowed @ data['config']['step']['user']['confirm_setup']. Got '[%key:common::config_flow::description::confirm_setup%]'

Error: Process completed with exit code 1.
```